### PR TITLE
Adds new outputStream configuration. (DAT-13144)

### PR DESCRIPTION
**Pull request by abrackx**
## Impact

- [ ] Bug fix (non-breaking change which fixes expected existing functionality)
- [x] Enhancement/New feature (adds functionality without impacting existing logic)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Description
Adds new outputStream command line configuration. Used to redirect all logging to STDOUT/STDERR.

## Things to be aware of
~This likely has some impact on maven usage, but I have not tested it.~ Turns out it doesn't because it's gated behind params that do not exist in maven.

### CODE REVIEW

File: liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java [hunk 1]
```diff
@@ -726,6 +728,27 @@ private void configureLogging(Level logLevel, String logFile, boolean wasDefault
...
+            StreamHandler streamHandler = new StreamHandler(logOutputStream.getOutputStream(), new SimpleFormatter());
+            streamHandler.setLevel(level);
+            setFormatterOnHandler(logService, streamHandler);
+            logger.addHandler(streamHandler);
+            switch (logOutputStream) {
+                case STDOUT:
+                    System.setErr(logOutputStream.getOutputStream());
+                case STDERR:
+                    System.setOut(logOutputStream.getOutputStream());
+            }
```
Comment (StevenMassaro) [inline liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java:748]: This should be a method on the enum class.

File: liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java [hunk 2]
```diff
@@ -726,6 +728,27 @@ private void configureLogging(Level logLevel, String logFile, boolean wasDefault
...
 
+    /**
+     * Configures the system output stream and error stream based on supplied global parameter.
+     */
+    private void setOutputStream(Logger logger, LogService logService) {
+        final LogOutputStream logOutputStream = LiquibaseCommandLineConfiguration.LOG_STREAM.getCurrentValue();
+        final Level level = LiquibaseCommandLineConfiguration.LOG_LEVEL.getCurrentValue();
+        if (logOutputStream != null) {
+            Arrays.stream(logger.getHandlers()).forEach(logger::removeHandler); // Unsure of impact of this statement.
+            StreamHandler streamHandler = new StreamHandler(logOutputStream.getOutputStream(), new SimpleFormatter());
```
Comment (StevenMassaro) [inline liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java:739]: What if one of the handlers you've removed on line 738 isn't a stream handler? You never put it back. Is that okay?

Comment (abrackx) [reply liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java]: Good point, looks like it works when I only remove handlers that are streamhandlers so I updated it to work that way. I'm unsure of the impact of removing all handlers.

File: liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java (hunk 1 above)
Comment (abrackx) [reply liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java]: Done!

**Review by StevenMassaro — APPROVED** [review]

**Review by filipelautert — APPROVED** [review]

File: liquibase-core/src/main/java/liquibase/logging/SystemOutputStream.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
+package liquibase.logging;
+
+import liquibase.util.StringUtil;
+
+import java.io.PrintStream;
+
+public enum SystemOutputStream {
```
Comment (suryaaki2) [inline liquibase-core/src/main/java/liquibase/logging/SystemOutputStream.java:7]: @abrackx Can you add javadoc comments to the class?

**Review by suryaaki2 — CHANGES_REQUESTED** [review]

**Comment by abrackx** [comment]
Closing in favor of new approach with new ticket.

**Pull request closed without merging**